spec/system/admin/order_cycles/complex_editing_multiple_updation_spec.rb

```diff
@@ -35,6 +35,9 @@
       click_link 'Orders'
     end
 
+    # Click dismiss on distributor warning
+    click_button 'Dismiss'
```

Comment:
Or we could setup the enterprise properly to avoid the warning?

Reply:
Done in #12652 :pray: